fastfood/shell.py

```diff
@@ -34,39 +33,18 @@
 _local = threading.local()
 LOG = logging.getLogger(__name__)
 NAMESPACE = 'fastfood'
-EXCLAIM = '\xe2\x9d\x97\xef\xb8\x8f'
-CHECK = '\xe2\x9c\x85'
-PIZZA = '\xf0\x9f\x8d\x95'
-INTERROBANG = '\xe2\x81\x89\xef\xb8\x8f'
-RED_X = '\xe2\x9d\x8c'
-
-
-def _fastfood_gen(args):
-
-    cookbook = book.CookBook(args.cookbook)
-    return food.update_cookbook(
-        cookbook, args.template_pack, args.stencil_set,
-        force=args.force, **args.options)
-
-
-def _fastfood_new(args):
-    written_files, cookbook = food.create_new_cookbook(
-        args.cookbook_name, args.template_pack, args.cookbooks)
-
-    if len(written_files) > 0:
-        print("%s: %s files written" % (args.cookbook_name,
-                                        len(written_files)))
-    else:
-        print("%s up to date" % args.cookbook_name)
-
-    return written_files, cookbook
+EXCLAIM = '\xe2\x9d\x97\xef\xb8\x8f'.decode('UTF-8')
+CHECK = '\xe2\x9c\x85'.decode('UTF-8')
+PIZZA = '\xf0\x9f\x8d\x95'.decode('UTF-8')
+INTERROBANG = '\xe2\x81\x89\xef\xb8\x8f'.decode('UTF-8')
+RED_X = '\xe2\x9d\x8c'.decode('UTF-8')
```

Comment:
It appears the problem was with L152 in shell.py:
`message.format...` won't work if message is non-Unicode.

I changed this:
```
            message = "{}  fastfood version {} uploaded {}\n"
            message = message.format(sym, vers, uploaded.ctime())
```
to this:
```
            message = u"{}  fastfood version {} uploaded {}\n"
            message = message.format(sym, vers, uploaded.ctime())
```

And that fixed it :)